misc/circuitsv2.json

```diff
@@ -11336,7 +11336,12 @@
         },
         {
           "FilterPath": [
-            "Object",
+            "Objects",
```

Comment:
This needs to be singular `"Object"` and not `"Objects"` or it will create another search filter.